src/helpers/authenticate.ts

```diff
@@ -65,7 +65,7 @@ export const customClaimContains = (
   value: string | boolean | number,
   child?: string
 ) =>
-  `auth.token.${claim}${child ? "." + child.replace(/\//, child) : ""} == ${s(
+  `auth.token.${claim}${child ? ("." + child.replace(/\//g, ".")).replace(/\.(\$.+?)(\.|$)/g, "[$1]$2") : ""} === ${s(
     value
```

Comment:
Why move to triple equals? All the examples I'd seen were double equals so I was concerned that the security parser was limited to that. I assume you've tested this?

Reply:
The examples I've seen use strong equality checks. Example here:

[https://firebase.google.com/docs/auth/admin/custom-claims]()